OpenPOS-APP/CheckoutOverview.xaml.cs

```diff
@@ -43,6 +43,9 @@ public async void OnClickedSplitPay(object sender, EventArgs args)
 	
    private async void OnClickedPay(object sender, EventArgs args)
    {
+       //TODO: REMOVE FOR TESTING
```

Comment:
This is fully build out in the other branch I would remove it so it doesn't mess with the other PR